spec.html

```diff
@@ -25238,34 +25256,34 @@ <h1>
             1. Assert: _element_ is a ClassStaticBlockDefinition Record.
             1. Append _element_ to _staticElements_.
         1. Set the running execution context's LexicalEnvironment to _env_.
-        1. Let _instanceExtraInitializers_ be a new empty List.
-        1. Let _staticExtraInitializers_ be a new empty List.
+        1. Let _instanceMethodExtraInitializers_ be a new empty List.
+        1. Let _staticMethodExtraInitializers_ be a new empty List.
         1. For each element _e_ of _staticElements_, do
           1. If _e_ is a ClassElementDefinition Record and _e_.[[Kind]] is not ~field~, then
-            1. Let _result_ be Completion(ApplyDecoratorsAndDefineMethod(_F_, _e_, _staticExtraInitializers_, *true*)).
+            1. Let _result_ be Completion(ApplyDecoratorsAndDefineMethod(_F_, _e_, _staticMethodExtraInitializers_, *true*)).
```

Comment:
If _e._[[Kind]] is ~accessor~, we should pass in `_e_.[[ExtraInitializers]]` instead of `_staticMethodExtraInitializers_`.

Ditto for the `_instanceMethodExtraInitializers_` in the next instance elements scan.